backend/services/syson-services/src/main/resources/DirectEdit.g4

```diff
@@ -29,7 +29,11 @@ package org.eclipse.syson.services.grammars;
 }
 
 expression :
-	name? multiplicityExpression? featureExpressions EOF
+	referenceExpression? name? multiplicityExpression? featureExpressions EOF
+;
+
+referenceExpression : 
+	'ref '
```

Comment:
Are you forced to use `'ref' ` instead of `'ref'` ?

Reply:
Yes, because we don't want to set the `isReference` feature for names like "refXXX".
The solution I found for that is to ensure that "ref" is followed by a space, to show the intent to set the isReference feature.
I can add the space after the 'ref' if you want.

Reply:
No, let's keep it like that, thank you.